include/ck/wrapper/operations/gemm.hpp

```diff
@@ -75,28 +92,29 @@ __device__ void blockwise_gemm_xdl(const ATensorType& a_local_tile_tensor,
     static_assert(is_same_v<DataType, typename ATensorType::TensorElementType>);
     static_assert(is_same_v<DataType, typename BTensorType::TensorElementType>);
 
-    using GemmAccDataType = std::
-        conditional_t<is_same_v<typename ATensorType::TensorElementType, int8_t>, int32_t, float>;
+    constexpr bool is_integer =
+        is_same_v<DataType, int8_t> || is_same_v<DataType, int16_t> || is_same_v<DataType, int32_t>;
```

Comment:
The standard library function: https://en.cppreference.com/w/cpp/types/is_integral should work here. Why not using it?

Reply:
Will be fixed with next PR